.github/workflows/ci.yml

```diff
@@ -43,6 +43,9 @@ jobs:
         uses: actions/setup-go@cdcb36043654635271a94b9a6d1392de5bb323a7 # v5.0.1
         with:
           go-version: stable
+          cache-dependency-path: |
+            go.sum
+            .github/.cache/buster-for-vars
```

Comment:
just curious why this is necessary and what is "buster" ?

Reply:
`setup-go` changed the caching mechanism and it's invalidating the cache for concurrent jobs. By adding these files we create unique caches for the job and by modifying the file we can "bust" the cache.

Reply:
It's not necessary for this PR though, I just wanted to speed it up and fix the cache. I can open a separate PR if it's a concern.